starter_code/components.js

```diff
@@ -0,0 +1,35 @@
+class Image extends React.Component{
+  constructor(){
+    super();
+    this.state = { image: "http://cdn2.business2community.com/wp-content/uploads/2016/01/QA-Danny-DeVito-008.jpg" }
+  }
+  _handleClick() {
+    if (this.state.image) {
+      this.setState({ image: !this.state.image });
+    } else {
+      this.setState({ image: this.state.image });
+    }
+  }
+  render(){
+    return(
+      <div>
+        <img src= { this.state.image } />
```

Comment:
Now down here, we need to use the `this.state.showImage` variable to determine whether or not to display our image. Something like:

```js
return(
  <div>
    {this.state.showImage ? <img src="http://cdn2.business2community.com/wp-content/uploads/2016/01/QA-Danny-DeVito-008.jpg" /> : null }
    <Button />
</div>
```

Here, we use a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) to decide whether or not to display the image, based on the current state of `this.state.showImage`.